website/docs/tutorials/07-testing/02-cheatcodes/declare.md

```diff
@@ -12,4 +12,5 @@ class DeclaredContract:
 ```
 Declares contract given a path relative to a Protostar project root.
 
-`config` is a keyword only argument kept for compatibility with the migration [declare cheatcode](../../06-deploying/02-migrations/declare.md). See related documentation for more information.
+- `contract` — A path to the **compiled** contract or a contract name. If you provide a contract name, Protostar will search for the compiled contract in the default build output (which is `build` in the root of the project) directory.
```

Comment:
[Documentation]
Testing cheatcodes don't use `ContractIdentifierResolver` hence this documentation is incorrect.